components/script/layout_dom/element.rs

```diff
@@ -787,7 +787,7 @@ pub struct ServoThreadSafeLayoutElement<'dom> {
 
     /// The pseudo-element type, with (optionally)
     /// a specified display value to override the stylesheet.
```

Comment:
> with (optionally) a specified display value to override the stylesheet.

There is not display value attached to `PseudoElement` (and there was none for  `PseudoElementType` either)as far as I can see, so we should remove this part of the comment

Reply:
Oh, good catch. I'll update the comment for this and for `node.rs.`